src/discord.ts

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+
+export default class Discord {
+  private baseUrl: string;
+  public accessAddress: string;
+  public signature: string;
+  public signatureData: any;
+
+  constructor(
+    baseUrl: string,
+    accessAddress: string,
+    signature: string,
+    signatureData: any
+  ) {
+    this.baseUrl = `${baseUrl}/discord`;
+    this.accessAddress = accessAddress;
+    this.signature = signature;
+    this.signatureData = signatureData;
+  }
+
+  setBaseUrl(baseUrl: string) {
+    this.baseUrl = `${baseUrl}/discord`;
+  }
+
+  connectDiscordWithApp(
+    appId: string,
+    discordServerId: string | number,
```

Comment:
Can it be both string and number ?

Reply:
discordServerId like this `989352128885686352`.   
I was confused as to whether it would come in as a number or as a string, but it seems to come in as a string when I bring an example. I'll fix it.